core/primitives/src/views.rs

```diff
@@ -204,14 +204,18 @@ pub struct StateItem {
     pub key: Vec<u8>,
     #[serde(with = "base64_format")]
     pub value: Vec<u8>,
-    pub proof: TrieProofPath,
+    /// Deprecated, always empty, eventually will be deleted.
+    // TODO(mina86): This was made deprecated at protocol version 56.  Remove it
+    // once we’re at 59.
```

Comment:
Also, I wrote this comment but then realised the structure is not borsh serialised so protocol version doesn’t matter here.  It’s more just time to get a chance for people to update.

```suggestion
    // TODO(mina86): This was deprecated in 1.30.  Get rid of the field
    // altogether at 1.33 or something.
```